### ISSUE TITLE
[BUG] FPS drops in certain worlds

### ISSUE THREAD

**Issue by grankless** [vigoren/foundryvtt-simple-calendar#285]
**Describe the bug**
Losing frames on both my end and players; performance is significantly impacted and tokens appear to skip around in movement. Issue has been identified using Find the Culprit and just general testing to happen while ony Simple Calendar is enabled. In one world, there are no issues and I stick to around ~60 fps; in another world, it tanks down to 30 and gets the performance issues; in my third game, it goes to around 45-50 FPS. I have not tested with others to see how performance is impacted in that one. In all worlds I have the same settings, with the only change from default being that I am using the Golarion calendar and I have disabled real time progression of the world clock.

I am unable to determine when this began as the game that has the ~30 fps issues had not been played for 3 weeks.
**Foundry Information**
 - Foundry Version: v9 build 249
 - Game System: PF2e 3.4.4
 - Simple Calendar Configuration: the only change from default being that I am using the Golarion calendar and I have disabled real time progression of the world clock.

**Additional context**

**Comment by vigoren** [comment]
So you have a large number of notes in these world's?

Or another module that is updating the world time frequently (for condition duration tracking or something like that)?

I am thinking it might be the first one and there could be a slow down with the check for which notes players want to be reminded of. This should be addressed in the next version.

**Comment by grankless** [comment]
Sorry for the long delay on response.  there were a alrge number of notes, though most weren't visible to players. I cleared some of the extraneous ones, and performance was the same. I have tested it with and without about-time active, which also did not impact performance. Fortunately I'm not too pressed about disabling Simple Calendar for however long it takes for an update to come out, this game is on a pretty small timeframe.
